sdk/servicebus/azure-servicebus/swagger/generate_attributes_sequence_tuple.py

```diff
@@ -0,0 +1,33 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+# This is a tool, not a part of the SDK code. Run it with Python 3.6+.
```

Comment:
I could be misremembering but I don't believe I saw a mention to this when reading that attribute map in the model module.  Typically I see a big "# === THIS IS AUTOGENERATED CODE, CHANGE AT YOUR OWN RISK, TOOLING LOCATED <here> ===" sort of flag at the start/end of the autogen block. (chose your favorite notation, just in terms of making that connectivity clear)

Reply:
Added a kind of disclaimer